code/__DEFINES/fonts.dm

```diff
@@ -14,3 +14,11 @@
 
 /// Emoji icon set
 #define EMOJI_SET 'icons/ui_icons/emoji/emoji.dmi'
+
+// Font metrics bitfield
+/// Include leading A width and trailing C width in GetWidth() or in DrawText()
+#define INCLUDE_AC (1<<0)
```

Comment:
I don't know what this means at all

Reply:
This is all from Lummox and I'm aware it has to be cleaned up. I wanted to get the PR up for the rest of it to be reviewed in the meantime.